src/utils/componentNameValidation.js

```diff
@@ -5,6 +5,10 @@ const allowedExtensions = {
     "py",
     "csv",
     "txt"
+  ],
+  "html": [
+    "html",
```

Comment:
don't know what's used in the projects, but might we want to support `.htm`?

Reply:
Unless it's relevant to the projects, I'd suggest avoiding showing that as a suggested extension to the user, to avoid confusion, considering they are beginner coders.

Reply:
to be fair, I don't think veteran coders use it either 😅 😆